src/main/java/DilemmaDetector/Consequences/ConsequenceContainer.java

```diff
@@ -30,29 +30,18 @@ public void saveConsequencesToOntology() {
         for (String decisionName: materialConsequencesByDecisionMap.keySet()){
             Set<Map.Entry<String, Double>> entries = materialConsequencesByDecisionMap.get(decisionName).entrySet();
             for(Map.Entry<String, Double> entityConsequence : entries){
-                Material_consequence material_consequence = factory.createMaterial_consequence(ObjectNamer.getName("material_consequence"));
-                material_consequence.addHas_material_value(entityConsequence.getValue().floatValue());
-                System.out.println("CONSEQ" + entityConsequence.getKey() + " "  + entityConsequence.getValue());
-                Entity entity = factory.getEntity(entityConsequence.getKey());
-                material_consequence.addMaterial_consequence_to(entity);
-//                /                saveMaterialConsequence(decisionName, material_consequence, entityConsequence.getKey(), entityConsequence.getValue());
-
+                saveMaterialConsequence(decisionName, entityConsequence.getKey(), entityConsequence.getValue());
             }
-
         }
     }
 
-    private void saveMaterialConsequence(String decisionName, Material_consequence consequence, String entityName, double value){
+    private void saveMaterialConsequence(String decisionName, String entityName, Double materialValue){
+        Material_consequence material_consequence = factory.createMaterial_consequence(ObjectNamer.getName("material_consequence"));
+        material_consequence.addHas_material_value(materialValue.floatValue());
         Decision decision = factory.getDecision(decisionName);
-        decision.addHas_consequence(consequence);
-        Set<Map.Entry<String, Double>> entries = materialConsequencesByDecisionMap.get(decisionName).entrySet();
-        for(Map.Entry<String, Double> entityConsequence : entries ){
-            Entity entity = factory.getEntity(entityName);
-//            System.out.println("MATERIAL CONSEQUENCES" + value + entity);
-            consequence.addHas_material_value((float)value);
-            consequence.addMaterial_consequence_to(entity);
-        }
-
+        decision.addHas_consequence(material_consequence);
```

Comment:
dla jednej decyzji ma być wiele konsekwencji materialnych rozumiem?

Reply:
Nie no, jedna powinna być chyba. Ale kod wygląda ok, może to jakieś pozostałości tylko.

Reply:
Tak przyjąłem że material consequence odnosi się do 1 obiektu i ma wartość straty w dolarach 
![image](https://user-images.githubusercontent.com/47008811/97677280-55163c00-1a92-11eb-81be-846b7a76e77b.png)

Reply:
Ale dlaczego material consequence odnosi się do 1 obiektu a nie do decyzji tak jak dotąd? Tak jak np mamy 1 killed czy injured consequence dla 1 decyzji.

Reply:
Hmm może faktycznie źle to zrobiłem. Pomyślałem że konsekwencje  jak killed są zawsze takie same bo zabity to zabity, a material consequences różnią się wartością szkody dla każdego obiektu

Reply:
Jak tak na to spojrzeć to faktycznie ma sens. Pytanie czy warto zmieniać taką koncepcję jaką już mamy. Nie niesie to żadnej dodatkowej informacji bo jak będziemy chcieli to sobie sprawdzimy co ile jest warte bo to jest w ontologii.

Reply:
Czyli myślisz żeby zrobić 1 material consequence do kilku samochodów, a w data propertisie wpisać sumę wszystkich szkód?

Reply:
Jak już robić tak jak zrobiłeś to utrzymując konwencję powinno być 1 material_consequence i wiele material_consequence_to w nim. (per decyzja oczywiście) Ale może wystarczy po prostu zapisać sumę ze wszystkich obiektów do material_consequence bo to jest to co nas naprawdę interesuje.

Reply:
Ok, to mogę tak przerobić